lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -233,3 +234,62 @@ function dosomething_kudos_get_kudos_for_user_by_file($fid) {
 
   return $kudos;
 }
+
+/**
+ * Check to see if the kudos term has been selected by the user on specified
+ * reportback.
+ *
+ * @param  [type] $id
+ * @param  object $reportbackItem
+ * @return bool
+ */
+function dosomething_kudos_term_is_selected($id, $reportbackItem) {
```

Comment:
Not suggesting as something for this PR, but I'd love to consider adopting our PSR-inspired code style in Phoenix as well, since it is can be pretty confusing switching back and forth from Dries-mode.